src/plugins/controls/public/time_slider/components/time_slider_popover_content.tsx

```diff
@@ -50,27 +41,65 @@ export function TimeSliderPopoverContent(props: Props) {
           };
         });
 
+  const {
+    useEmbeddableDispatch,
+    useEmbeddableSelector: select,
+    actions: { setIsAnchored },
+  } = useReduxEmbeddableContext<TimeSliderReduxState, typeof timeSliderReducers>();
+  const dispatch = useEmbeddableDispatch();
+  const isAnchored = select(getIsAnchored);
+  const rangeInput = isAnchored ? (
+    <AnchoredRange
+      value={props.value}
+      onChange={props.onChange}
+      stepSize={props.stepSize}
+      ticks={ticks}
+      timeRangeMin={props.timeRangeMin}
+      timeRangeMax={props.timeRangeMax}
+    />
+  ) : (
+    <SlidingWindowRange
+      value={props.value}
+      onChange={props.onChange}
+      stepSize={props.stepSize}
+      rangeRef={props.rangeRef}
+      ticks={ticks}
+      timeRangeMin={props.timeRangeMin}
+      timeRangeMax={props.timeRangeMax}
+    />
+  );
+  const anchorStartToggleButtonLabel = isAnchored
+    ? i18n.translate('controls.timeSlider.settings.unanchorStartSwitchLabel', {
+        defaultMessage: 'Unpin start',
+      })
+    : i18n.translate('controls.timeSlider.settings.anchorStartSwitchLabel', {
+        defaultMessage: 'Pin start',
+      });
```

Comment:
**Nit:** Also inline with our style guide, we typically separate out all of our internationalization strings into a separate file for components rather than doing it inline - for example, for the options list control, we have `../options_list/components/options_list_strings.ts` :+1: